### PR TITLE
add new community hub image and hero

### PR DESCRIPTION
**Pull request by corwintines**
## Description

- Adds community hub image
- Adds HeroHub implementation to community page

### CODE REVIEW

**Review by pettinarip — APPROVED** [review]
LGTM

Would wait for @nloureiro's approval before merging.

**Review by nloureiro — APPROVED** [review]

**Comment by nloureiro** [comment]
> LGTM
> 
> Would wait for @nloureiro's approval before merging.

approved!

**Pull request merged**
